espresso/mx.espresso/suite.py

```diff
@@ -290,6 +295,11 @@
                         "toolchain": "sulong:SULONG_BOOTSTRAP_TOOLCHAIN",
                     },
                 },
+                "linux-musl": {
+                    "<others>": {
+                        "ignore": "GNU Linux-only",
```

Comment:
While i understand not using the eden library on musl, it can't work to not build mokapot.
If espresso is not supported on musl, i would prefer to keep that a build failure for now.

Reply:
I've removed changes to this file, so building espresso now fails. I'm going to quickly look into the failure and see if I can come up with a fix in a day or two. If not, then I'll produce a separate PR later